__tests__/app.test.js

```diff
@@ -249,3 +249,53 @@ describe("POST api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH api/articles/article_id", () => {
+  test("200: returns article with updated votes", () => {
+    const newVotes = {
+      inc_votes: 10,
+    };
+    return supertest(app)
+      .patch("/api/articles/1")
+      .send(newVotes)
+      .expect(200)
+      .then((res) => {
+        const { article } = res.body;
+
+        expect(article).toHaveProperty("article_id", expect.any(Number));
+        expect(article).toHaveProperty("title", expect.any(String));
+        expect(article).toHaveProperty("topic", expect.any(String));
+        expect(article).toHaveProperty("author", expect.any(String));
+        expect(article).toHaveProperty("body", expect.any(String));
+        expect(article).toHaveProperty("created_at", expect.any(String));
+        expect(article).toHaveProperty("votes", expect.any(Number));
```

Comment:
Does this test prove that the article votes have updated? Any its currently expecting any number